admin/app/indexes/ContentApiTagsEnumerator.scala

```diff
@@ -39,7 +37,7 @@ object ContentApiTagsEnumerator extends Logging {
   }
 
   def enumerateTagType(tagType: String) = enumeratePages { page =>
-    getResponse(ContentApiClient.tags.tagType(tagType).pageSize(MaxPageSize).page(page))
+    contentApiClient.getResponse(contentApiClient.tags.tagType(tagType).pageSize(MaxPageSize).page(page))
```

Comment:
I don't see the issue here. It's pretty much the same as before, it's just that removing the import makes it more visible.
Did I miss something?